bg-jobs/Dockerfile

```diff
@@ -0,0 +1,7 @@
+ARG  DOCKER_REPOSITORY
+ARG  TEACHER_TRAINING_API
+ARG  CODE_VERSION=${GIT_BRANCH:-latest}
+
+FROM ${DOCKER_REPOSITORY}/${TEACHER_TRAINING_API}:${CODE_VERSION}
+
+CMD bundle exec rails db:migrate && bundle exec sidekiq -q save_statistic
```

Comment:
Can we create a generic container which subscribes to all the queues?
Or just have one queue?

Reply:
I'd propose we do that work as a separate PR, once we have this one up and running and stable. We can then have a single PR to retire the other two containers and allow this one to process all the queues.

Reply:
OK makes sense too